Formula/p/prestodb.rb

```diff
@@ -3,8 +3,8 @@ class Prestodb < Formula
 
   desc "Distributed SQL query engine for big data"
   homepage "https://prestodb.io"
-  url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-server/0.288.1/presto-server-0.288.1.tar.gz", using: :nounzip
-  sha256 "e04e54add9d776587daebda0cac8806173018f170e8fd018f60e0d12ec5184ed"
+  url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-server/0.289/presto-server-0.289.tar.gz", using: :nounzip
```

Comment:
```suggestion
  url "https://repo1.maven.org/maven2/com/facebook/presto/presto-server/0.289/presto-server-0.289.tar.gz", using: :nounzip
```

Reply:
OK, the previous url is connecting to the maven central or mirrors.
https://central.sonatype.org/search/example-urls

Somehow the url is dead, and got 403 error.